scripts/patch_release/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM sagebionetworks/genie:version-16.4.0
```

Comment:
Guessing this will need to be updated every time we have a new patch release?

Reply:
^ I dont think so, I think we can probably lock it down to a specific version of the GENIE package.

Reply:
Oops, confused myself because I kept thinking about our package releases matching consortium releases. 

Makes sense. It will probably have to be updated when we want to incorporate new features from a package